scripts/globals/mobskills/Immortal_shield.lua

```diff
@@ -0,0 +1,28 @@
+----------------------------------
+-- Immortal Shield
+--
+-- Description: Grants a Magic Shield effect for a time.
+-- Type: Enhancing
+--
+-- Range: Self
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+
```

Comment:
Unnecessary newline

Reply:
actually he's missing the return that should be here

Reply:
You planning to come back to this PR @hooksta4 ? without the return line this script will error out

Reply:
Yes, started a new job and been doing shift work. Will get to this as soon as I can.